website/docs/using-odo/json-output.md

```diff
@@ -0,0 +1,43 @@
+---
+title: JSON Output
+sidebar_position: 2
+---
+# JSON Output
+
+The `odo` commands that output some content generally accept a `-o json` flag to output this content in a JSON format, suitable for other programs to parse this output more easily.
+
+The output structure is similar to Kubernetes resources, with `Kind`, `apiVersion`, `metadata` ,`spec` and `status` fields.
+
+List commands return a `List` resource, containing an `items` (or similar) field listing the items of the list, each item being also similar to Kubernetes resources.
+
+Delete commands return a `Status` resource; see the [Status Kubernetes resource](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/status/).
+
+Other commands return a resource associated with the command (`Application`, `Storage`', `URL`, etc).
+
+The exhaustive list of commands accepting the `-o json` flag is currently:
+
+| commands                       | Kind (version)                          | Kind (version) of list items                                 | Complete content?         | 
+|--------------------------------|-----------------------------------------|--------------------------------------------------------------|---------------------------|
+| odo application describe       | Application (odo.dev/v1alpha1)          | *n/a*                                                        |no                         |
+| odo application list           | List (odo.dev/v1alpha1)                 | Application (odo.dev/v1alpha1)                               | ?                         |
```

Comment:
What's the difference between "n/a" and "?" ?

Reply:
n/a (not applicable) indicates that this information (Kind (version) of list items) if not applicable, because the discussed resource is not a list

"?" is a request for comment from the team, to discuss if the content is useful and complete, or if we should work on it